docs/appendix/changelog.md

```diff
@@ -19,6 +19,13 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog date="2024-06-24" color="changed" title="Viam Agent roaming mode" %}}
+
+The [Viam Agent](/fleet/provision/) now supports [roaming mode](/fleet/provision/#use-roaming-mode). You can use the Viam Agent in roaming mode to allow it to attempt to connect to each specified WiFi network in your configuration.
```

Comment:
What is now called "Roaming Mode" was previously the default/only mode of operation. The NEW mode is actually also the new default, single-network.